apps/test/unit/templates/sectionProgressV2/IconKeyTest.jsx

```diff
@@ -1,41 +1,60 @@
 import React from 'react';
 import {fireEvent, render, screen} from '@testing-library/react';
 import {expect} from '../../../util/reconfiguredChai';
+import * as utils from '@cdo/apps/utils';
+import sinon from 'sinon';
 import IconKey from '@cdo/apps/templates/sectionProgressV2/IconKey';
 
 describe('IconKey Component', () => {
   it('renders the open state initially', () => {
+    sinon.stub(utils, 'tryGetLocalStorage').returns('true');
     render(<IconKey isViewingValidatedLevel={false} expandedLessonIds={[]} />);
     expect(screen.getByLabelText('Icon key')).be.visible;
     expect(screen.queryByText('More Details')).be.visible;
     expect(screen.getByText('Assignment Completion States')).to.exist;
     expect(screen.getByText('Teacher Actions')).to.exist;
```

Comment:
```suggestion
    screen.getByText('Teacher Actions');
```
nit: We don't need the `expect` if we are just checking existence of a single item via a `getBy` query. This will also print out a much better error without the expect. I'd recommend removing all `expects().to.exist` in the file

Reply:
Thanks for this context!  I made this change!